test/com/linkedin/drelephant/tuning/AlertingTest.java

```diff
@@ -45,33 +46,31 @@ private void testDeveloperAlerting() {
 
     assertTrue("No data within the range provided ", notificationData.size() == 0);
 
-
-
     JobSuggestedParamSet jobSuggestedParamSet = JobSuggestedParamSet.find.select("*").where().findUnique();
     jobSuggestedParamSet.updatedTs = new Timestamp(startTime + 100);
     jobSuggestedParamSet.createdTs = new Timestamp(endTime+1-259200000);
+    jobSuggestedParamSet.paramSetState = JobSuggestedParamSet.ParamSetStatus.EXECUTED;
```

Comment:
It will be better to write separate test case , instead of modifying the earlier one